roles/pulp3/tasks/configure.yml

```diff
@@ -19,3 +19,11 @@
       force: no
 
   become: true
+
+- block:
```

Comment:
Is there going to be more than one command?
If not, why make it a block?

Reply:
There will not be other commands. removing the block. thanks for the suggestion.